Makefile

```diff
@@ -57,10 +45,9 @@ packages: rpm srpm
 
 # Python packaging
 wheel:
-	python setup.py bdist_wheel
-	python3 setup.py bdist_wheel
+	hatch build
 upload:
-	twine upload dist/*.whl
+	hatch publish
```

Comment:
This should probably have 'tarbal' and 'wheel' as dependency, or some other way to make sure we have files to publish.